index.html

```diff
@@ -0,0 +1,393 @@
+                        <div class="col-12 drop-down-menu">
+                            
+                            <ul>
+                                <li>
+                                    <a href="about.html">About</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="close_button">&times;</span>
+                        </div>
+                    </div>
+                </div>
+
+
+            </div>
+            <div class="row">
+
+                <div class="container">
+                    <div class="row">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-11">
+                            <h1 class="main_header">African women in tech group <br> launches to support diversity</h1>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-11 main_paragraph">
+                            <p>
+                                There is a significant gap of representation of women in the technology field in Africa, and having a support group in place that focuses on African women can be very beneficial to fill this gap.
+                            </p>
+                            <p>This group can also be a great platform to empower and inspire young women to pursue careers in technology and to become role models, by highlighting the successes and contributions of African women in the field, and providing access to mentorship and networking opportunities with successful female tech leaders</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-7 bottom-header">
+                            <p>
+                                <span class="header-paragraph">2023.10.15(THU)-16(FRI)</span>
+                                <br> @African women in tech group launches to support diversity and inclusion.</p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </div>
+
+        <div>
```

Comment:
- [ Required ] Please consider changing this tag to the HTML5 semantic `section` tag, the `section` tag is usually used to wrap logical sections of a page that have `hx` tags, since this is a logical section of your page please wrap it in a `section` tag as shown below. 
```HTML
<section class="container">
  <div class="row">
     <div class="col-1 desk-div"></div>
       <div class="col-11">
           <h1 class="main_header">African women in tech group <br> launches to support diversity</h1>
        </div>
   </div>
</section>
```